### PR TITLE
Imported users mailing

### PR DESCRIPTION
**Pull request by jaleszek**
Send reset password email after import.

### CODE REVIEW

File: app/mailers/users/import_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Users::ImportMailer < ApplicationMailer
+  def notify_about_release(user_id, raw_token)
+    user = User.find(user_id)
+    @token = raw_token
+    @user = user
+    mail to: user.email, subject: "The Queen’s Awards for Enterprise: Welcome to our new application system"
```
Comment (matthewford) [inline app/mailers/users/import_mailer.rb]: Line is too long. [108/100]

**Comment by jaleszek** [comment]
please review someone ?

File: app/mailers/users/import_mailer.rb (hunk 1 above)
Comment (matthewford) [inline app/mailers/users/import_mailer.rb]: Queen`’`s weird char - use `'`

File: app/views/users/import_mailer/notify_about_release.html.slim [hunk 2]
```diff
@@ -0,0 +1,31 @@
+p style="font-size: 19px; line-height: 1.315789474; margin: 30px 0 30px 0;"
+  ' Dear
+  - if @user.full_name.present?
+    = @user.full_name
+  - else
+    = @user.email
+  ' ,
+
+p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;"
+  ' Firstly, we’d like to thank you for your past interest in The Queen’s Awards for Enterprise.
```
Comment (matthewford) [inline app/views/users/import_mailer/notify_about_release.html.slim]: same - replace these char with `'` not `’ `

**Comment by jaleszek** [comment]
merge me I'm so cool : - )

**Pull request merged**